nusight2/src/client/components/robot_selector/style.css

```diff
@@ -69,5 +69,5 @@
     0px 0px 3px 0px rgba(0, 0, 0, 0.15),
     0px 3px 8px 0px rgba(0, 0, 0, 0.15);
   overflow: hidden;
-  width: 100%;
+  width: 400px;
```

Comment:
We could get away with making this a little more compact.

<details>
<summary>Compare before/after</summary>

<img src="https://user-images.githubusercontent.com/5924865/126255967-ee4302b9-021a-4348-9f9f-9502c927f900.png" alt="Screenshot of selector panel before and after changes" width="260px">

</details>

```suggestion
  width: 330px;
```

Reply:
@ysims wanted a bit of a bigger gap hence the change but I can do this instead

Reply:
The template areas can be bigger than the parent size, and will be drawn as if the parent was that large.

In this case it's better to do `180px 1fr`, to make the first column 180px and have the second fill the remaining space.

Reply:
Gotcha okay